addons/account/models/account_analytic_line.py

```diff
@@ -35,7 +35,6 @@ def on_change_unit_amount(self, cr, uid, id, prod_id, quantity, company_id,
             return {}
         product_obj = self.pool.get('product.product')
         analytic_journal_obj = self.pool.get('account.analytic.journal')
-        product_price_type_obj = self.pool.get('product.price.type')
```

Comment:
why copy-paste business stuff in v7 and v8 methods of  on_change_unit_amount ? can you simply  it ?